router/schema.go

```diff
@@ -0,0 +1,46 @@
+package main
+
+import (
+	"github.com/flynn/flynn/Godeps/_workspace/src/github.com/flynn/go-sql"
+	"github.com/flynn/flynn/pkg/postgres"
+)
+
+func migrateDB(db *sql.DB) error {
+	m := postgres.NewMigrations()
+	m.Add(1,
+		`CREATE EXTENSION IF NOT EXISTS "uuid-ossp"`,
+		`CREATE EXTENSION IF NOT EXISTS "hstore"`,
+		`CREATE FUNCTION set_updated_at_column() RETURNS TRIGGER AS $$
+	BEGIN
+		NEW.updated_at = CURRENT_TIMESTAMP AT TIME ZONE 'UTC';
+		RETURN NEW;
+	END;
+$$ language 'plpgsql'`,
+		`CREATE TYPE route_type AS ENUM ('http', 'tcp')`,
+		`CREATE TABLE routes (
+	route_id uuid PRIMARY KEY DEFAULT uuid_generate_v4(),
+	parent_ref text NOT NULL,
+	type route_type NOT NULL,
+	config hstore NOT NULL,
```

Comment:
The bools & number values keep their type on the way out thanks to `hstore_to_json_loose`. The values are stored as unescaped strings, so that function produces the same JSON going at as going in.